pages/treeherder.py

```diff
@@ -61,6 +61,7 @@ class ResultSet(PageRegion):
 
         _datestamp_locator = (By.CSS_SELECTOR, '.result-set-title-left > span a')
         _jobs_locator = (By.CLASS_NAME, 'job-btn')
+        _pin_all_jobs_locator = (By.CSS_SELECTOR, '.result-set .glyphicon-pushpin')
```

Comment:
Changing this to By.CLASS_NAME generates this error locally:
E       InvalidSelectorException: Message: The given selector .result-set .glyphicon-pushpin is either invalid or does not result in a WebElement. The following error occurred:
E       InvalidSelectorError: Compound class names not permitted

Were you able to get it to pass this way?